app/assets/stylesheets/govuk_publishing_components/components/_chart.scss

```diff
@@ -47,3 +47,14 @@
     padding: 0 govuk-spacing(4);
   }
 }
+
+.gem-c-chart .gem-c-chart__a11y-note-link {
+  display: none;
+}
+
+.js-enabled {
+  .gem-c-chart .gem-c-chart__a11y-note-link {
+    display: inline-block;
+    margin-bottom: govuk-spacing(2);
```

Comment:
A really tiny point, but this margin might be better on the rule above, rather than introducing it as part of a state change?

Reply:
Thanks @matthillco - good point, I have now made that change 👍